pkg/systemd/logs.html

```diff
@@ -33,14 +33,16 @@
     <div class="content-header-extra">
       <div class="filters">
         <div class="filter">
-            <label for="journal-current-day-menu" translate>Time</label>
-            <select id="journal-current-day-menu" class="ct-select">
-              <option value="recent" translate>Recent</option>
-              <option value="boot" translate>Current boot</option>
-              <option value="previous-boot" translate>Previous boot</option>
-              <option value="last-24h" translate>Last 24 hours</option>
-              <option value="last-week" translate>Last 7 days</option>
-            </select>
+            <button id="log-filters" class="btn btn-default dropdown-toggle ct-select" type="button" data-toggle="dropdown">
+              <span translate>Filters</span>
+              <div class="caret"></div>
+            </button>
+            <ul class="dropdown-menu" id="logs-predefined-filters">
```

Comment:
I thought `<select>` had better a11y all around, even with tabindex? What was the reason for going back to a list?

Reply:
select is a different component. It has currently selected value, this is something different - it is bunch of button that are rolled up once you click on the button.

Reply:
From IRC: We now need to allow not selecting anything.